notus/scanner/utils.py

```diff
@@ -127,26 +129,23 @@ def init_logging(
     log_file: Optional[str] = None,
     foreground: Optional[bool] = False,
 ):
-    rootlogger = logging.getLogger()
-    rootlogger.setLevel(log_level)
+    root_logger = logging.getLogger()
+    root_logger.setLevel(log_level)
 
     formatter = logging.Formatter(
         f"%(asctime)s {name}: %(levelname)s: (%(name)s) %(message)s"
     )
     if foreground:
-        console = logging.StreamHandler()
-        logfile = WatchedFileHandler(log_file)
-        console.setFormatter(formatter)
-        logfile.setFormatter(formatter)
-        rootlogger.addHandler(console)
-        rootlogger.addHandler(logfile)
-    elif log_file:
-        logfile = WatchedFileHandler(log_file)
-        logfile.setFormatter(formatter)
-        rootlogger.addHandler(logfile)
-    else:
-        syslog = SysLogHandler("/dev/log")
-        syslog.setFormatter(formatter)
-        rootlogger.addHandler(syslog)
-
-    return rootlogger
+        console_handler = logging.StreamHandler()
+        console_handler.setFormatter(formatter)
+        root_logger.addHandler(console_handler)
+    if log_file:
+        log_file_handler = WatchedFileHandler(log_file)
+        log_file_handler.setFormatter(formatter)
+        root_logger.addHandler(log_file_handler)
+    if not foreground and not log_file:
```

Comment:
```suggestion
    if log_file:
        log_file_handler = WatchedFileHandler(log_file)
        log_file_handler.setFormatter(formatter)
        root_logger.addHandler(log_file_handler)
    elif not foreground and not log_file:
```

I think this could be an `elif` ... so you might skip one condition check ... 🚤

Reply:
Nope. That needs to stay as it is. If foreground is set console is activated. If log file is set the logs are stored in the log file. If both are set both console and log file are used. Only if both are not set syslog should be used.

Reply:
Yeah. But I think that is still working. It checks if `foreground` is set. In both cases it checks if `log_file` is set afterwards. And if `log_file` is set, it won't check if `foreground` and `log_file` are not set, since `log_file` is set anyways. But nevermind. ^^

Reply:
ah got it. yeah you are right :-)